setup.py

```diff
@@ -42,6 +43,7 @@
             "pubsub_lifecycle_change = microcosm_pubsub.conventions:LifecycleChange",
             "pubsub_send_batch_metrics = microcosm_pubsub.metrics:PubSubSendBatchMetrics",
             "pubsub_send_metrics = microcosm_pubsub.metrics:PubSubSendMetrics",
+            "resource_cache = microcosm_pubsub.caching:configure_resource_cache",
```

Comment:
We may want to extract this resource caching stuff and other helpers into some `microcosm-caching` for a more standalone solution we can use also in non-daemon use cases. WDYT @GabeLambda  @pelegdvir ?

Reply:
I definitely want us to invest in a shared caching library at some point. Let's start using it in a few places first before that, though.

Reply:
I like the idea.
Check our own `lethe` service